julia/.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Should this be included? 
I kind of think that unless there is something lang specific it shouldn't be included, since it's not in the other examples. (unless you want to put an `.envrc` in all the examples?)

Reply:
Yeah, I think your right. I'll get rid of it.